xml/ay_bigfile.xml

```diff
@@ -8984,64 +8984,818 @@ config:type="boolean"&gt;true&lt;/keep_install_network&gt;</screen>
   </sect1>
 
 
-  <sect1 xml:id="Configuration.Security.users">
-   <title>Users</title>
+  <sect1 xml:id="Configuration.Security.users_and_groups">
+   <title>Users and Groups</title>
 
    <para>
-    The &rootuser; and at least one normal user can be added during
-    install using data supplied in the control file. User data and passwords
-    (encrypted or in clear text) are part of the
-    <literal>configure</literal> resource in the control file.
+    AutoYaST supports defining local users, groups, special login settings and even
+    default options for new users. Those settings are defined in the following sections:
    </para>
 
-   <para>
-    At least the &rootuser; should be configured during auto-installation
-    so you can login after the installation is finished. It will also ensure
-    nobody else can login to the system (in case the password is not set).
-   </para>
+   <variablelist>
+    <varlistentry>
+     <term>users</term>
+     <listitem>
+      <para>
+       List of users
+      </para>
+     </listitem>
+    </varlistentry>
+    <varlistentry>
+     <term>user_defaults</term>
+     <listitem>
+      <para>
+       Default options for new users
+      </para>
+     </listitem>
+    </varlistentry>
+    <varlistentry>
+     <term>groups</term>
+     <listitem>
+      <para>
+       List of groups
+      </para>
+     </listitem>
+    </varlistentry>
+    <varlistentry>
+     <term>login_settings</term>
+     <listitem>
+      <para>
+       Special login settings like password-less login or autologin
+      </para>
+     </listitem>
+    </varlistentry>
+   </variablelist>
 
-   <para>
-    The two users in the following example are added during system
-    configuration.
-   </para>
+   <note>
+    <title>Users and groups set up during the first stage</title>
+    <para>
+     Starting with &productname; 12 SP1, users and groups are set up during the
```

Comment:
I would mention the the openSUSE part too. As far I am remembering  there are special tags for SLES and openSUSE. So we could distinguish between the two cases and produces different descriptions.